packages/fx-core/src/component/generator/openApiSpec/const.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/**
+ * @author yuqzho@microsoft.com, Ning Tang
+ */
+
+export const defaultDeclarativeCopilotManifestFileName = "declarativeAgent.json";
+export const defaultDeclarativeCopilotActionId = "action_1";
+export const copilotPluginExistingApiSpecUrlTelemetryEvent = "copilot-plugin-existing-api-spec-url";
```

Comment:
copilotPlugin is also an old naming, change it to declarativeAgent

Reply:
Better not to update the event name to avoid bi issues